### PR TITLE
Add jobs for origin-web-console test

### PR DESCRIPTION
**Pull request by jwforres**

### CODE REVIEW

**Comment by jwforres** [comment]
Depends on https://github.com/openshift/origin-ci-tool/pull/126

@stevekuznetsov the old console job was test_pull_requests (plural) and these are test_pull_request (singular) I went for consistency with the other job definitions. I assume there is no issue with changing the name other than telling Rosie to do something different?

cc @spadgett

**Comment by jwforres** [comment]
Also FYI I was testing with prefixed job names zzz_test_pull_request_origin_web_console and zzz_test_branch_origin_web_console while i was getting things working. So we'll need to clean up those jobs.

**Comment by stevekuznetsov** [comment]
> I assume there is no issue with changing the name other than telling Rosie to do something different?

Yes -- I chose the singular as it's more appropriate -- we are testing one singular pull request. See information about updating Rosie config here:  https://github.com/openshift/ci-config#configuring-test-pull-requests

> Also FYI I was testing with prefixed job names zzz_test_pull_request_origin_web_console and zzz_test_branch_origin_web_console while i was getting things working. So we'll need to clean up those jobs.

Ack

**Comment by jwforres** [comment]
@stevekuznetsov the successful run based on the updated AMI is here https://ci.openshift.redhat.com/jenkins/view/All/job/test_pull_request_origin_web_console/2/

ready to merge this?

File: sjb/config/test_cases/test_pull_request_origin_web_console.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+parent: 'test_cases/test_branch_origin_web_console.yml'
+overrides:
```
Comment (stevekuznetsov) [inline sjb/config/test_cases/test_pull_request_origin_web_console.yml]: To make this Rosie-compatible for `[test]` you will need to add:

```yml
overrides:
  test: 'origin-web-console'
```

As per [this doc](https://github.com/openshift/aos-cd-jobs/blob/master/sjb/syntax.md#merge-and-test).

**Comment by jwforres** [comment]
@stevekuznetsov updated, re-running test job here https://ci.openshift.redhat.com/jenkins/view/All/job/test_pull_request_origin_web_console/3/console

**Comment by stevekuznetsov** [comment]
Not sure what Rosie will do there, but this part LGTM -- next would be pushing this up and changing the Rosie config

**Pull request merged**